app/javascript/theme/shared/_form-input.scss

```diff
@@ -6,4 +6,10 @@
   label {
     margin-bottom: 0.25rem;
   }
+
+  &__instructional-copy {
+    color: $neutral70;
+    margin: 0.5rem 0;
```

Comment:
So here would be `margin-top: 0.5rem`, since we want it spaced from the input it's with, but not to push space below the `FormInput`.  Then, since we _do_ want a little extra space between the login submit button and the form input, we'd bump the `Login__button { margin-top: 1rem; }` declaration to 1.5rem.  

That'd get you your extra 0.5rem without making it present anytime we use that instructional-copy class.

Reply:
got it! just fixed :)